pkg/util/trace/impl/motrace/report_statement.go

```diff
@@ -276,5 +277,10 @@ var ReportStatement = func(ctx context.Context, s *StatementInfo) error {
 	if !GetTracerProvider().IsEnable() {
 		return nil
 	}
+	// Filter out the MO_LOGGER SQL statements
+	if s.User == sqlWriter.MOLoggerUser {
```

Comment:
i believe cut off the suffix of statement text is better than drop the record.
may result like: `insert into system.statement_info valus (...)`

Reply:
will review the statementinfo collected in the following steps